src/app/playlists/page.tsx

```
@@ -86,6 +132,49 @@ export default function PlaylistsPage() {
+              </div>
+              <form onSubmit={handleCreatePlaylist}>
+                <input
+                  type="text"
+                  value={newPlaylistTitle}
+                  onChange={(e) => setNewPlaylistTitle(e.target.value)}
+                  placeholder="プレイリスト名"
+                  className="w-full bg-zinc-800 border border-zinc-700 rounded-xl px-4 py-3 text-white placeholder-zinc-500 focus:outline-none focus:ring-2 focus:ring-white/20 mb-6"
+                  autoFocus
+                />
+                <div className="flex justify-end gap-3">
+                  <button
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)